packages/tools/src/eventListeners/mouse/mouseDownListener.ts

```diff
@@ -9,6 +9,8 @@ import getMouseEventPoints from './getMouseEventPoints';
 const { MOUSE_DOWN, MOUSE_DOWN_ACTIVATE, MOUSE_CLICK, MOUSE_UP, MOUSE_DRAG } =
   Events;
 
+const DOUBLE_CLICK_TOLERANCE_MS = 400;
```

Comment:
What is the best way to make this configurable?

Reply:
I guess we can worry about this in another PR, since we are not configuring [any other suff](https://github.com/cornerstonejs/cornerstone3D-beta/blob/feat/sphere-surface-cut/packages/tools/src/init.ts#L26)